data/hai/hai news.txt

```diff
@@ -0,0 +1,251 @@
+
+
+# Visual suggestion: glasses, clean-cut attire, possibly scientific accoutrements
+news "trading in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Merchant captain"
+	portrait
+		portrait/human02
+		portrait/human04
+		portrait/human06
+		portrait/human10
+		portrait/human15
+		portrait/human17
+		portrait/human31
+		portrait/human58
+		portrait/human63
+		portrait/human72
+		portrait/human73
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Hai Tech is incredible compared to ours. Every time I add a ship to my fleet, I always make a trip up here to stock up on Hai tech."
+			"Have you tried out one of those Hai anti-missile systems? They'll waste any missile in Human space."
+			"Can you believe these Hai? They're light-years ahead of us, right under our noses."
+			"Have you visited the Quarg ringworld here in Hai space? It's a humbling sight. There's probably more Quarg in there than there are humans anywhere. Makes you feel pretty insignificant."
+			"Did you see the stuff they sell on Icelake? Hai technology is amazing."
+			"I met some young Hai pilots. It sounds like they start out as poor as we do."
+			"I sold my entire stock of food at this port. I guess even Hai get tired of nuts."
+			"Some Hai were protesting the work week. They want to work only 32 hours a week instead of 40. I used to work 70 hours a week for the Syndicate!"
+
+
+# Visual suggestion: glasses, clean-cut attire, possibly scientific accoutrements
+news "science in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Scientist"
+			"Engineer"
+	portrait
+		portrait/human13
+		portrait/human37
+		portrait/human43
+		portrait/human64
+		portrait/human71
+		portrait/human83
+	message
+		word
+			"A while back, my Hai research partner was nearing a breakthrough, and then she vanished. I haven't seen her since. She probably joined the Unfettered."
+			"Can you believe these Hai? They're light-years ahead of us, right under our noses."
+			"Have you visited the Quarg ringworld here in Hai space? It's a humbling sight. There's probably more Quarg in there than there are humans anywhere. Makes you feel pretty insignificant."
+			"It's amazing how effective Hai fertilizer has become. If we had that technology, hunger would be a thing of the past."
+			"Hai can terraform planets a hundred times faster than we can. I wonder why they didn't do that at Darkwaste?"
+			"Hai have the most amazing holographic technology. One of them put on a backpack holoprojector. When he turned it on, he looked like an elephant. The legs and tail even tracked his movements! I wonder if they can mimic humans too?"
+			"The Hai have an amazing grasp of quantum chromodynamics. It's why their shield generators are so effective."
+			"The alloys Hai use in their ships are way beyond our technology."
```

Comment:
This blurb feels pretty short compared to the rest. Maybe add a bit extra?

Reply:
Now, "The alloys Hai use in their ships are way beyond our technology. They're twice as strong with half the weight. No wonder their ships are so fast."